src/ClassicUO.Client/LegionScripting/doc.md

```
@@ -0,0 +1,1279 @@
+
+
+<details>
+<summary><h3>UseObject(serial, skipQueue)</h3></summary>
+
+*Attempt to use(double click) an object.*
+
+**Parameters**  
+| Name | Type | Optional | Description |
+| --- | --- | --- | --- |
+| serial | uint | No | The serial |
+| skipQueue | bool | Yes | Defaults true, set to false to use a double click queue |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)